app/src/main/java/com/it/hungvt/movieapp/utils/Constant.java

```diff
@@ -0,0 +1,18 @@
+package com.it.hungvt.movieapp.utils;
+
+/* Define constants variable */
+
+import android.support.annotation.IntDef;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
```

Comment:
ERROR: Unused import: java.lang.annotation.RetentionPolicy. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports